models/README.md

```diff
@@ -76,40 +76,456 @@ Multi-label sequence classification for 10 sensitive information categories
 ### References
 Well-Read Students Learn Better: On the Importance of Pre-training Compact Models, 2019,  https://arxiv.org/abs/1908.08962
 
-## Phishing Email Detection
-### Model Overview
-Phishing email detection is a binary classifier differentiating between phishing/spam and non-phishing/spam emails and SMS messages.
-### Model Architecture
-BERT-base uncased transformer model
-### Training
-Training consisted of fine-tuning the original pretrained [model from google](https://huggingface.co/bert-base-uncased). The labeled training dataset is around 5000 SMS messages from a public dataset- [SMS Spam Collection](https://archive.ics.uci.edu/ml/datasets/SMS+Spam+Collection)
-### How To Use This Model
-This model is an example of customized transformer-based phishing email detection. It can be retrained for specific detection needs and customized the emails of your enterprise using the training scripts in the repo.
-#### Input
-Entire email as a string
-#### Output
-Binary sequence classification as phishing/spam or non-phishing/spam
-### References
-- https://archive.ics.uci.edu/ml/datasets/SMS+Spam+Collection
-- Devlin J. et al. (2018), BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding
-https://arxiv.org/abs/1810.04805
+# Phishing Detection
 
+# Model Overview
+
+## Description:
+Phishing detection is a binary classifier differentiating between phishing/spam and benign emails and SMS messages. <br>
+
+## References(s):
+1. https://archive.ics.uci.edu/ml/datasets/SMS+Spam+Collection <br>
+2. Devlin J. et al. (2018), BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding https://arxiv.org/abs/1810.04805 <br> 
+
+## Model Architecture: 
+**Architecture Type:** Transformers <br>
```

Comment:
Overall comment: Can we add a bullet point (* ) to all the answers in this model card (if they do not already have one)?
It'll help the readability of the model card (and will make it prettier in .md).